src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  justify-content: space-between;
+  padding-right: 50px;
+
+  &__logo {
+    display: flex;
+    margin-left: 50px;
+  }
+}
+
+.nav {
+  &__tech-list {
+    display: flex;
+  }
+
+  &__category-link {
+    font-weight: 500;
+    font-size: 12px;
+    color: $primary-color;
+    padding: 23px 0;
+    text-decoration: none;
+    display: block;
+    text-transform: uppercase;
+    position: relative;
```

Comment:
Order those css peoperties too. There should be order similar to this: position, display, padding and only then everything about fonts.